src/commit-message.ts

```diff
@@ -0,0 +1,45 @@
+import { PullRequestInfo } from './models'
+import { Config } from './config'
+
+export function getCommitMessage (
+  pullRequestInfo: PullRequestInfo,
+  config: Config
+) {
+  return config.mergeCommitMessage
+    ? getCustomCommitMessage(pullRequestInfo, config.mergeCommitMessage)
+    : null
+}
+
+export function splitCommitMessage (message: string) {
+  const [title, ...body] = message.split('\n')
```

Comment:
Should we circumvent extra newlines when a configuration has:

```
{title}

{description}
```
instead of
```
{title}
{description}
```

Reply:
Fixed in https://github.com/bobvanderlinden/probot-auto-merge/commit/ef4f77d5a129f49b7e1784d419489ef914b4e3f9